requirements/base.txt

```diff
@@ -1,6 +1,6 @@
 pyrsistent~=0.16.0; python_version<"3"
 boto3~=1.5
 enum34~=1.1; python_version<"3.4"
-jsonschema~=3.0
-six~=1.11
+jsonschema~=3.2
```

Comment:
have we verified that this does not break any tests?

Reply:
These changes do not do anything in fact. `~=3.0` and `~=3.2` both spec to `3.x`